docs/Extension/guides/backup-and-restore.md

```diff
@@ -0,0 +1,109 @@
+---
+sidebar_label: '🔏 Backup and Restore'
+sidebar_position: 3
+description: 'Extension Support: Backup and Restore'
+---
+
+# Backup and Restore
+
+<div class="video-container">
+<iframe width="560" height="315" src="https://www.youtube.com/embed/hyXIKU66cvc?si=maJqk5iY8N9uLY9B" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+</div>
+
+The Backup and Restore feature of the Universal Profiles Browser Extension allows users to export all of their UPs to a password-protected file to be restored later.
+
+:::info Available Recovery Options
+
+Other safe options to restore your Universal Profile:
+
+- Adding an email or 2FA recovery feature via my.universalprofile.cloud
+- Setting another wallet as a recovery controller, known as third-party wallet recovery
+
+:::
+
+## Accessing Backup and Restore
+
+Navigate to the extension settings and locate the Backup and Restore section.
+
+<div style={{display: 'flex', gap: '10px', justifyContent: 'center', alignItems: 'flex-start', flexWrap: 'wrap'}}>
+  <img src="/img/extension/backup-and-restore/up-settings.png" alt="Extension Settings" style={{maxHeight: '500px', width: 'auto', maxWidth: '45%', minWidth: '280px', objectFit: 'contain'}} />
+  <img src="/img/extension/backup-and-restore/bnr section.png" alt="Backup and Restore Section" style={{maxHeight: '500px', width: 'auto', maxWidth: '45%', minWidth: '280px', objectFit: 'contain'}} />
+</div>
+
+## 1. Backup Profiles
+
+This feature lets you back up all Universal Profiles (UPs) currently loaded in your browser extension, for both LUKSO Mainnet and Testnet.
```

Comment:
I had users who were surprised to see that the Backup file included all the Universal Profiles (they thought they would backup only one UP). Suggest mentioning this specific aspect of the backup in a yellow or blue callout